test/customer_repository_test.rb

```diff
@@ -1,15 +1,27 @@
-require_relative '../lib/customer_repository'
-require_relative 'test_helper'
+require_relative "../lib/customer_repository"
+require_relative "test_helper"
 
 class CustomerRespositoryTest < Minitest::Test
-
   attr_reader :sales_engine, :customer_repo, :data
 
-
   def setup
-    @data = [{id: 45, first_name: "bob", last_name: "jones", created_at: "2010-01-01", updated_at: "2014-01-01" },
-      {id: 46, first_name: "jane", last_name: "Jones", created_at: "2012-01-01", updated_at: "2015-01-01" },
-      {id: 47, first_name: "may", last_name: "johnson", created_at: "2011-01-01", updated_at: "2014-01-01" }]
+    @data = [
+             { id: 45,
```

Comment:
Use 2 spaces for indentation in an array, relative to the start of the line where the left bracket is.